### PR TITLE
BC1–53-feat:Creating the initial structure of the project showland

### PR DESCRIPTION
**Pull request by sadiGFG**

### CODE REVIEW

File: show-land/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <section class="footer-section">footer section</section>
+</div>
+
+</main>
+    
+</div>
+</div>
+
+    </body>
+    </html>
```
Comment (ahmadmohammadirad2006) [inline show-land/index.html:45]: Please use "prettier" to format the code as mentioned in code convention's page :)

File: show-land/sass/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+
+/*------------------------------------*\
+  #COLOR
+\*------------------------------------*/
+$colorPrimary:#37b24d;
+$colorSecondary:#364fc7;
+$colorTertiary:#6741d9;
+$colorFourth:#343a40;
+$colorIfth:#1864ab;
```
Comment (ahmadmohammadirad2006) [inline show-land/sass/_variables.scss:9]: I think you meant 
```
$colorFifth: #1864ab;
```
So if it is a typo, fix it please

File: show-land/index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    <section class="super-fast-section">super fast</section>
+    <section class="Home-page-section">Home page </section>
+    <section class="page-builder-section">page builder section</section>
+    <section class="benefits-section">Benefits section </section>
+    <section class="features-section ">features section </section>
+    <section class="powerful-features-section">Powerful features section</section>
+    <section class="help-users-section">Help users section</section>
+    <section class="trusted-section ">trusted section </section>
+    <section class="footer-section">footer section</section>
+</div>
```
Comment (ahmadmohammadirad2006) [inline show-land/index.html:37]: Which tag does this </div> tag close?

File: show-land/index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+</main>
+    
+</div>
+</div>
```
Comment (ahmadmohammadirad2006) [inline show-land/index.html:42]: This one like the previous one does not have an opening tag.

**Review by ahmadmohammadirad2006 — CHANGES_REQUESTED** [review]

File: show-land/sass/main.scss [hunk 5]
```diff
@@ -0,0 +1,6 @@
+@import"variables";
+
+@import"top-nav";
+@import"typography";
+@import"../../shared/scss/components/button";
+@import"../../shared/scss/base/reset";
```
Comment (ahmadmohammadirad2006) [inline show-land/sass/main.scss:6]: Please separate the imports into logical parts like you have done for your variables file

**Review by MaxRichard2007 — CHANGES_REQUESTED** [review]

**Review by ahmadmohammadirad2006 — APPROVED** [review]

File: show-land/sass/_variables.scss [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+/*------------------------------------*\
+  #COLOR
+\*------------------------------------*/
+$colorPrimary:#37b24d;
+$colorSecondary:#364fc7;
+$colorTertiary:#6741d9;
+$colorFourth:#343a40;
+$colorFifth:#1864ab;
+
+$colorWihte: #fff;
```
Comment (MichaelAndish) [inline show-land/sass/_variables.scss:11]: there is a typo, $colorWhite

File: show-land/sass/main.scss [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+/*------------------------------------*\
+  #BASE
+\*------------------------------------*/
+@import "../../shared/scss/base/reset";
+
+/*------------------------------------*\
+  #COMPONENTS
+\*------------------------------------*/
+@import "../../shared/scss/components/button";
+@import "components/button";
```
Comment (MichaelAndish) [inline show-land/sass/main.scss:21]: You imported shared button and your button! which one do you want to use?

**Review by MichaelAndish — CHANGES_REQUESTED** [review]

File: show-land/sass/main.scss [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+/*------------------------------------*\
+  #BASE
+\*------------------------------------*/
+@import "../../shared/scss/base/reset";
+
+/*------------------------------------*\
+  #COMPONENTS
+\*------------------------------------*/
+@import "../../shared/scss/components/button";
```
[the remark this answers is not in the document]
Comment (sadiGFG) [reply show-land/sass/main.scss:20]: Your opinion is respected, but for now it is better to be like this

**Review by MichaelAndish — APPROVED** [review]

**Pull request merged**